seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json

```diff
@@ -340,7 +340,8 @@
         "</div>"
       ],
       "tests": [
-        "assert(code.match(/val.imageLink/gi), 'message: You should have used the <code>imageLink</code> property to display the images.');"
+        "assert(code.match(/val.imageLink/g), 'message: You should use the <code>imageLink</code> property to display the images.');",
+        "assert(code.match(/html\\s*(\\+=|=\\s*html\\s*\\+\\s*)\\s*(\"|')\\s*<\\s*img\\s+src\\s*=\\s*(?:\"\\s*'|'\\s*\")\\s*\\+\\s*val\\.imageLink\\s*\\+\\s*(?:\"\\s*'|'\\s*\")\\s*>\\s*(?:\"|')\\s*;\\s*/gi), 'message: You should use a valid HTML img tag.');"
```

Comment:
> this only displays the last image.
No. It display all the images after you click `Get Message` Button.

Is that what you are saying? or i am missing something?